torch/fft/__init__.py

```diff
@@ -889,10 +893,10 @@
 
     >>> f = torch.fft.fftfreq(4)
     >>> f
-    tensor([ 0.0000,  0.2500,  -0.5000, -0.2500])
+    tensor([ 0.0000,  0.2500, -0.5000, -0.2500])
 
-    >>> torch.fftshift(f)
-    tensor([-0.5000, -0.2500, 0.0000, 0.2500])
+    >>> torch.fft.fftshift(f)
+    tensor([-0.5000, -0.2500,  0.0000,  0.2500])
```

Comment:
Found these discrepancies from running the doctests manually. So, it seems fft doctests aren't being run in CI.

Reply:
Nice fix.